src/Sylius/Behat/Page/Shop/Checkout/AddressingPage.php

```diff
@@ -124,6 +176,8 @@ protected function getDefinedElements()
             'shipping_country' => '#sylius_shop_checkout_addressing_shippingAddress_countryCode',
             'shipping_city' => '#sylius_shop_checkout_addressing_shippingAddress_city',
             'shipping_postcode' => '#sylius_shop_checkout_addressing_shippingAddress_postcode',
+            'login_password' => 'input[type=\'password\']',
+            'login_button' => 'a.button.labeled',
```

Comment:
We definitely need more specific selectors prefixed with ``sylius-``. Please have a look at my recent changes to shop pages.